### DISCUSSION TITLE
Migration to Native SDK

### DISCUSSION THREAD

**Discussion by swarali-nandgaonkar** [AxisCommunications/acap-computer-vision-sdk-examples#200] [Q&A] [answered]
I already have applications developed using the cv-sdk for the cameras. How to migrate to the nativ sdk?

**Comment by pataxis** [comment, marked as answer]
Hi @swarali-nandgaonkar , thanks for your question.

You could start out trying to build and install the [Hello World](https://axiscommunications.github.io/acap-documentation/docs/get-started/hello-world-example.html) example to get a feel of how native ACAP applications (.eap-files) work.

From there you could continue to the [Native APIs](https://axiscommunications.github.io/acap-documentation/docs/api/native-sdk-api.html) and look at the ones of interest, there should be at least one example showing usage of each API. The majority of the examples are written in C.

I hope this can give you an idea of how a migration can be done. I understand that moving from Python to C/C++ can be a bit of a challenge.

Note that the ACAP Computer Vision solution (container support) will be supported on AXIS OS 12.x for products with SDK architecture `aarch64`, so that's two years of migration for the AXIS OS active track and then 5 additional years on the 2026 LTS track. We updated the information banner after there was some misunderstanding of the first version.
